packages/browser/src/ClientAuthentication.ts

```diff
@@ -54,12 +55,12 @@ export default class ClientAuthentication {
     private environmentDetector: IEnvironmentDetector
   ) {}
 
-  private urlOptionToUrl(url?: URL | string): URL | undefined {
+  private urlOptionToUrl(url?: UrlParse | string): UrlParse | undefined {
```

Comment:
This function is still badly named - but hopefully it'll disappear completely if we can get rid of `URLParse` - so no point in suggesting a better name right now (like `convertToUrlParse()` !).